infrastructure/BoothRepository.cs

```diff
@@ -1,41 +1,60 @@
 ﻿using Core.Domain;
 using Core.Entity;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace infrastructure
 {
     public class BoothRepository : IRepository<Booth>
     {
+        private readonly BazarContext _ctx;
+
+        public BoothRepository(BazarContext ctx)
+        {
+            _ctx = ctx;
+        }
+        // Counts the amount of booths in database.
         public int Count()
         {
-            throw new NotImplementedException();
+            return _ctx.Booth.Count();
         }
-
+        // Creates Booth
         public Booth Create(Booth entity)
         {
-            throw new NotImplementedException();
+            // Making variable for when we have to hash password etc. 
```

Comment:
This comment does not make so much sense and you should not make things you do not need yet